pkg/OWNERS

```diff
@@ -0,0 +1,4 @@
+files:
+  Makefile:
+    owners:
```

Comment:
Do we need this extra level? Can't we just hang the teams right off the pattern?

Reply:
I was thinking we could potentially add more attributes, e.g.

```
files:
   Makefile:
      owners:
       - security
      always_review: true # do not allow PRs to merge unless it has been reviewed by security, but flexible on this
```

Reply:
if we think that's too complicated though, happy to take it out